submissions/shramkoweb/js-dom/main.css

```diff
@@ -0,0 +1,73 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    padding: 0;
+    font-size: 14px;
+    line-height: 20px;
+    font-family: "Open Sans", "Arial", sans-serif;
+}
+
+img {
+    max-width: 100%;
+    display: block;
+    height: auto;
+}
+
+.main {
+    display: grid;
+    grid-template-columns: 200px 1fr;
+}
+
+.content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.frameworks-list {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    padding: 0;
+}
+
+.content img {
+    max-height: 200px;
+}
+
+.frameworks-list li {
+    margin-bottom: 10px;
+}
+
+.button {
```

Comment:
Why do you use `user-select: none;` here? I'd recommend adding `cursor: pointer;` for this button as well.

Reply:
Because I don't want select text in button. 
It about user select property